app/app.go

```diff
@@ -842,6 +890,7 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	paramsKeeper.Subspace(poolincentivestypes.ModuleName)
 	paramsKeeper.Subspace(superfluidtypes.ModuleName)
 	paramsKeeper.Subspace(gammtypes.ModuleName)
+	paramsKeeper.Subspace(wasm.ModuleName)
```

Comment:
@ethanfrey 
The pinned contracts are not loaded within the `if loadLatest` [block](https://github.com/CosmWasm/wasmd/blob/master/app/app.go#L676-L679)

Reply:
Very good catch. Thank you